### PR TITLE
Create view and controller for Terms of Participation page

### PR DESCRIPTION
**Pull request by brennaksamuel**

### CODE REVIEW

File: app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/terms.scala.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.incometaxsubscriptioneligibilityfrontend.config.AppConfig
+@import uk.gov.hmrc.play.views.html.helpers.form
+
+@(postAction:Call)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@main_template(title = Messages("terms.title")) {
```
Comment (AlexRimmerHMRC) [inline app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/terms.scala.html]:
```suggestion
@main_template(title = messages("terms.title")) {
```
To keep the use of messages consistent in the view

File: app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/terms.scala.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+@(postAction:Call)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@main_template(title = messages("terms.title")) {
+
+    <h1 class="heading-large">@messages("terms.heading")</h1>
+
+    <div class="form-group">
+        <p>@messages("terms.line1")</p>
+            <ul class="bullets">
```
Comment (AlexRimmerHMRC) [inline app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/terms.scala.html]:
```suggestion
            <ul class="list list-bullet">
```
This change will make the bullet points spread out more than they currently are, as per the prototype

Comment (AlexRimmerHMRC) [reply app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/terms.scala.html]: Also formatting of the file, ul tag should be tabbed to the same amount of the p tag

File: it/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/controllers/TermsControllersISpec.scala [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+package uk.gov.hmrc.incometaxsubscriptioneligibilityfrontend.controllers
+
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import org.scalatestplus.play.guice.GuiceOneServerPerSuite
+import uk.gov.hmrc.incometaxsubscriptioneligibilityfrontend.utils.{ComponentSpecBase, ViewSpec}
+import uk.gov.hmrc.incometaxsubscriptioneligibilityfrontend.assets.MessageLookup.{Terms => messages}
+import play.api.test.Helpers._
+
+class TermsControllersISpec extends ComponentSpecBase with ViewSpec with GuiceOneServerPerSuite {
```
Comment (VLukovski) [inline it/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/controllers/TermsControllersISpec.scala]:
```suggestion
class TermsControllerISpec extends ComponentSpecBase with ViewSpec with GuiceOneServerPerSuite {
```

**Pull request merged**